app/src/components/CompetitionHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Typography } from "@mui/material";
+import { Trans, useTranslation } from "react-i18next";
+
+export const CompetitionHeader = ({
+  name,
+  registrationOpen,
+  registrationClose,
+  series,
+}: {
+  name: string;
+  registrationOpen: Date;
+  registrationClose: Date;
+  series: boolean;
+}) => {
+  const { t } = useTranslation();
+
+  const currentDate = new Date();
+  const isRegistrationOpen = currentDate > registrationOpen;
```

Comment:
Don't we need to check it's also less than registration close? Or is this meant to be `hasRegistrationOpened`/`didRegistrationOpen`/something to that effect?